pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -787,7 +787,8 @@ func TestFilterPlugins(t *testing.T) {
 			if err != nil {
 				t.Fatalf("fail to create framework: %s", err)
 			}
-			status := f.RunFilterPlugins(context.TODO(), nil, pod, nil)
+			statusMap := f.RunFilterPlugins(context.TODO(), nil, pod, nil)
```

Comment:
we should add a test for `runAllFilters`

Reply:
exist in framework_test.go

Reply:
oh, then we should be checking that the statuses here are what we expect, additionally to checking the merge.

Reply:
done.